BracketeerCDT/src/com/chookapp/org/bracketeer/cdt/ClosingBracketHintVisitor.java

```diff
@@ -457,8 +457,8 @@ private void addBrackets(IASTName name) throws BadLocationException
         } 
         else if( name instanceof ICPPASTQualifiedName)
         {
-            IASTName[] names = ((ICPPASTQualifiedName) name).getNames();
-            for (IASTName n : names)
+            ICPPASTNameSpecifier[] names = ((ICPPASTQualifiedName) name).getQualifier();

```

Comment:
Looks like this method was retired here:

commit 4b31af20b150204afcc22e15333226b41dea77eb
Author: Sergey Prigogin <eclipse.sprigogin@gmail.com> 2016-03-16 21:56:26
Committer: Sergey Prigogin <eclipse.sprigogin@gmail.com> 2016-03-16 21:58:43
Parent: f1944d2d7f450d04c4aa4fcb8ff3b9d9e31cd8db (Marked deprecated APIs with @noreference tags to be able to remove them later.)
Branches: fix-bug-509751, fix-bug-519311, fix-bug-519473, master, origin/cdt_9_0, origin/cdt_9_1, origin/cdt_9_2, origin/cdt_9_3, origin/cdt_9_4, origin/cdt_9_5, origin/containerbuild, origin/master

Removed deprecated ICPPQualifiedName.getNames() method.

Change-Id: I6e84cba6c56fe5467205ef85e24035e6b2e86f50

Reply:
Like Bracketeer and I'm willing to help with the maintenance. I started to contribute to the Eclipse CDT as well.